apps/dot-voting/app/components/VoteDetails/index.js

```diff
@@ -109,11 +108,12 @@ const VoteDetails = ({ vote, onVote }) => {
                 voteWeights={voteWeights}
                 votingPower={votingPower}
               />
-            ) :(
+            ) : (
               <VotingResults
                 vote={vote}
                 options={vote.data.options}
                 voteWeights={voteWeights}
+                balance={(type === 'allocation') ? BigNumber(vote.data.balance).div(BigNumber(10 ** decimals)).toNumber() : 0}
```

Comment:
No need for this prop because `VotingResults` already has access to `vote.data.type` and `vote.data.balance`.